src/resourceGathering/ResourceGatheringBuilder.java

```diff
@@ -31,7 +32,7 @@ public Context build(Context<Object> context) {
 				ContinuousSpaceFactoryFinder.createContinuousSpaceFactory(null);
 		ContinuousSpace<Object> space =
 				spaceFactory.createContinuousSpace("space", context,
-						new RandomCartesianAdder<Object>(),
+						new SimpleCartesianAdder<Object>(),
```

Comment:
So was there actually any benefit to changing this into a simple cartesian adder?